.github/workflows/ci.yml

```diff
@@ -32,13 +32,13 @@ jobs:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
 
-    - name: Update npm
-      if: matrix.node-version == '6.x'
-      run: npm install -g npm@^5
-
     - name: Install dependencies
       run: npm install
 
+    - name: Install jest for node < 14
+      if: matrix.node-version == '10.x' || matrix.node-version == '12.x'
+      run: npm install jest@^27.5.1
```

Comment:
Perfecto

Glad to have you around, I went through this all manually on my machine(s) as well but you're on top of it! very nice